apps/dashboard/test/system/files_test.rb

```diff
@@ -68,10 +68,16 @@ class FilesTest < ApplicationSystemTestCase
       # if this fails it is due to the directory table not reloading
       # assert_selector '#directory-contents tbody tr', count: 3, wait: 10
       assert_selector 'tbody a', exact_text: 'app', wait: MAX_WAIT
-
+      
+      # we need to wait due to the nature of how the "% copy files" displays
+      assert_selector 'span', text: 'copy files', count: 1, wait: MAX_WAIT 
```

Comment:
Sorry to do this to you, but these are going to conflict with #1974. I found out what the issue with these tests are.